src/function.rs

```diff
@@ -59,6 +60,14 @@ impl Function {
   }
 }
 
+fn abs_path(context: &FunctionContext, path: &str) -> Result<String, String> {
+  let corrected_path = context.invocation_directory.join(path);
```

Comment:
Is this necessary? I saw that `path_exists` is just called directly on the path var.

Reply:
Good point, `path_exists` has a bug, it needs to be made relative to `search.working_directory`. This should also be joined with search.working_directory.

Reply:
Changed!